workflow/python/covid19_scrapers/states/delaware.py

```diff
@@ -14,7 +14,14 @@
 
 
 class Delaware(ScraperBase):
-    DATA_URL = 'https://myhealthycommunity.dhss.delaware.gov/locations/state'
+    '''
+    Delewares scraper has an acceptable use agreement that must be submitted before
```

Comment:
sp: Delaware
Let's summarize the agreement here, too: `acceptable use agreement prohibiting attempting to reidentify individuals` and maybe say that we do not do that.

Reply:
yikes and 😆 at the spelling. and ok, that sounds way better. will fix!